bin/eslint-formatters-calypso/src/lib/parsing-errors.js

```diff
@@ -0,0 +1,36 @@
+/* eslint strict: "off" */
+
+'use strict';
+
+module.exports = function( report ) {
+	// deep clone the report, so we can create a new one
+	// to tweak and edit in place - as this function remains pure.
+	let newReport = [];
+
+	const isParsingError = ( message ) => {
+		if ( ( message.ruleId === null ) &&
+		( message.severity === 2 ) &&
+		( message.message.startsWith( 'Parsing error:' ) ) ) {
```

Comment:
I have upgraded my knowledge about this and changed the check to only look for the `fatal` key.